### PR TITLE
Fix issue with url_to_slug method not handling scp-style git urls correctly

### LINKED ISSUE #8: Unable to use plugin with git@github.com:owner/repo.git repository URL format
global environment hook fails with 
```
Traceback (most recent call last):
--
  | File "/usr/local/bin/bk-ssm-secrets", line 11, in <module>
  | load_entry_point('bk-ssm-secrets==0.5.0', 'console_scripts', 'bk-ssm-secrets')()
  | File "/usr/local/lib/python3.7/site-packages/bk_ssm_secrets-0.5.0-py3.7.egg/bk_ssm_secrets/__main__.py", line 14, in main
  | File "/usr/local/lib/python3.7/site-packages/bk_ssm_secrets-0.5.0-py3.7.egg/bk_ssm_secrets/helpers.py", line 66, in url_to_slug
  | ValueError: Invalid URL scheme found: git@github.com:<orgname>/<reponame>.git
  | 🚨 Error: Error setting up bootstrap: The global environment hook exited with status 1
```

### PR DESCRIPTION
**Pull request by vgrigoruk**
Fixes #8

### CODE REVIEW

File: bk_ssm_secrets/tests/test_helpers.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+class TestHelpers(unittest.TestCase):
+    def test_url_to_slug(self):
+        assert(helpers.url_to_slug("https://github.com/a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("https://github.com/a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("git@github.com:a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("user@github.com:a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("https://git@github.com/a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("ssh://git@github.com/a/b.git") == "github.com_a_b.git")
+        assert(helpers.url_to_slug("ssh://git@github.com:22/a/b.git") == "github.com-22_a_b.git")
+        assert(helpers.url_to_slug("https://github.com:8080/a/b.git") == "github.com-8080_a_b.git")
```
Comment (mikeknox) [inline bk_ssm_secrets/tests/test_helpers.py:13]: thanks @vgrigoruk ,
Does this handle URI's with the form: `https://github.com:8080/~a/b.git` etc ?
The `~` was replaced with `_` originally.

Comment (vgrigoruk) [reply bk_ssm_secrets/tests/test_helpers.py:13]: Pushed a commit that fixes this :)

**Comment by xiaket** [comment]
Hi @vgrigoruk , thanks again for your help! Can I suggest we add the `parsed.scheme` check back in, and use this regex to handle that special case, while in most cases we still use urlparse result to form the URL?

**Comment by vgrigoruk** [comment]
> `parsed.scheme` check back in, and use this regex to handle that special case, while in most cases we still use urlparse result to form the URL?

Sure, I'll push another commit soon

**Review by mikeknox — APPROVED** [review]
thanks

**Pull request merged**